composer.json

```diff
@@ -79,6 +79,7 @@
         "drupal/group": "^1.4",
         "drupal/groupmedia": "^2.0@alpha",
         "drupal/honeypot": "^2.0",
+        "drupal/hreflang": "^1.10",
```

Comment:
Why is this change included in this PR? The addition of hreflang belongs to PGOV-743 whose commits aren't even in this branch.

Reply:
@RichardDavies I merged in master and those weird changes are gone now. I think I somehow merged in the commits from master without updating the head of my branch, so GitHub thought they were part of my PR. 🤷